docs/decex/staking.md

```diff
@@ -24,51 +24,59 @@ NPoS allows for virtually all SSC holders to participate, thus maintaining high
+Operators must stake an amount higher than this domain's minimum stake for a right to participate in the execution and earn execution fees. An operator's chances to become a slot leader are directly proportional to the percentage of their stake against the total amount staked by all operators of this domain. As such, operators are incentivized to recruit nominators to increase their stake. This means that each domain operator stake acts essentially as a pool for nominators. When registering as an operator, each operator specifies their minimum nominator stake and nomination tax of their pool. The nomination tax is a percentage that the operator collects on all fees earned by executing blocks, before they are shared with nominators, a commission for the operator's work. The tax amount is automatically restaked as part of the operator's stake.
 
-Any SSC token holder who has more than the minimum nominator stake may choose to join this operator’s pool by submitting the nomination extrinsic with the deposit amount of SSC they wish to stake. 
+Any SSC token holder who has more than the minimum nominator stake (currently 1 SSC) may choose to join this operator’s pool by submitting the nomination extrinsic with the deposit amount of SSC they wish to stake. 
 
-1. The amount of deposited SSC is added to the list of pending transfers within the operator's pool. 
-2. At the end of an epoch (currently 100 blocks), the nominator is awarded their shares in the pool. The stake shares are the percentage of the total stake that is allocated to each nominator. The stake shares are used to calculate the share of the operator's rewards that the nominator is entitled to based on the amount they have staked and for how long. The stake shares are calculated as follows:
-    1. Compute the operator’s pool end-of-epoch $\text{shares\_per\_ssc}$ as the total number of shares divided by the sum of all stake in the pool and rewards gained during the previous epoch 
+1. The amount of deposited SSC is added to the list of pending deposits within the operator's pool. 
+2. At the end of an epoch (currently 100 blocks, ~ 10 minutes), the nominator's deposit is processed.
+3. A part of the deposit is taken as a reserve towards a storage fee fund. This reserve is calculated as a percentage of the deposit (currently, 20%), and is used to pay for the storage fees of bundles created by the operator of the pool and does not affect the stake distribution. The reserved amount is transferred to the operators storage fee fund, while the rest of the deposit remains locked in the nominator's account. This amount is partially refunded with each withdrawal. 
```

Comment:
```suggestion
3. A part of the deposit is taken as a reserve towards a storage fee fund. This reserve is calculated as a percentage of the deposit (currently, 20%), and is used to pay for the storage fees of bundles created by the operator of the pool and does not affect the stake distribution. The reserved amount is transferred to the operator's storage fee fund, while the rest of the deposit remains locked in the nominator's account. This amount is partially refunded with each withdrawal. 
```

(assuming the storage fee fund is not shared by all operators?)

Reply:
If I try to misunderstand these updates it's not clear why we need the storage fee reserve. My question would be "why are the storage fees not being charged to the user who submits the transaction?"

Reply:
You're right, I'll make this more clear. The storage fees are charged to the user, but only after the tx is executed on domain, however consensus chain requires front payment for storage and that's where the pool's storage fund comes in. Each operator pool has a separate fund.